### DISCUSSION TITLE
Kitty Image Protocol replying "EFBIG: Too much data\"

### DISCUSSION THREAD

**Discussion by fortnider** [kovidgoyal/kitty#6562] [Q&A] [answered]
Hello, I am currently attempting to write a program that displays an image in the terminal using Kitty's image protocol. However, when I print out the graphics escape code, the terminal replies with "_Gi=10;EFBIG: Too much data\". Tried looking it up and could not find any information. What does this mean and how do I fix this? My payload is only 48 bytes (64 in base64) so I don't understand why it's giving me this error. 

The string that I am printing out is the following: 
> "\x1b_Gf=24,s=4,v=4,a=T,i=10,t=d;MTAxMDAxMDAxMDExMDAwMDEwMTExMDEwMTEwMDExMTAxMTAxMDAwMTExMDAxMDEwMTEwMDEwMDExMTAwMTAxMTExMDAwMDAwMTAwMTExMTExMDEwMTAwMDEwMTAxMTEwMTExMDEwMTExMTEwMTEwMTExMTAxMTEwMTEwMDExMTExMTAxMTAwMDExMDExMDAwMTEwMDExMTAxMTAxMDExMTExMDExMDExMTExMDExMDExMTEwMTExMDExMTAxMTExMTAxMTEwMDAxMDExMTEwMTExMDAwMDAwMTEwMTEwMTExMTAxMTEwMTExMTAwMDAwMTEwMTEwMTExMTAxMTEwMDExMTAwMDAwMTAxMTEwMTExMDExMTEwMDExMDAwMDAwMTAwMTAxMDAxMDAxMTAwMTEwMTAwMDAxMTEwMDAwMTAxMTAwMTAxMDExMDEwMDEwMTAxMTEwMDExMTAwMDAxMDExMDAxMDExMTAwMDExMDAxMDAxMDAxMDEwMDExMDEx\x1b\\"

**Comment by kovidgoyal** [comment, marked as answer]
Because you are sending more data than the 3 * 4 * 4 = 48
You are sending 384 bytes of data.

**Reply by fortnider** [reply]
Thank you for the quick reply. I was converting a string of binary to base64 instead of converting the actual binary to base64. I feel quite silly now.
